aptos-move/framework/aptos-framework/sources/conversions.move

```diff
@@ -0,0 +1,30 @@
+// Provides a framework for converting well-known types to and from bytes
+module aptos_framework::conversions {
+    use aptos_framework::util;
+    use std::bcs;
+
+    public fun to_address(bytes: vector<u8>): address {
+        util::from_bytes(bytes)
```

Comment:
Is this by extend what one would expect? (That is some lower or higher endian of the addresses in the bytes?)

Reply:
s/extend/accident

Reply:
There's no magic here. An address is by convention 32 bytes, we're not adding convenience libraries that might be in Rust for ... UX reasons.

Reply:
> We can't ship this like this:
> 
> * The functionality belongs into the aptos-stdlib
> * The naming in a general `convert` module should be not `to_address` it should be `vector_to_address`. (I showed the signatures the other day.)
> 
> Lets not in the last minute add some half-thought out APIs to the frameworks.

With respect to `vector_to_address` -- vector of what? The name should be very explicitly clear. So maybe byte_conversions would be an appropriate module since everything is about converting to and from bytes. Thoughts?